modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/header/APIMANAGER3614DuplicateTransferEncodingHeaderTestCase.java

```diff
@@ -0,0 +1,84 @@
+/*
+*Copyright (c) 2010-2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*WSO2 Inc. licenses this file to you under the Apache License,
+*Version 2.0 (the "License"); you may not use this file except
+*in compliance with the License.
+*You may obtain a copy of the License at
+*
+*http://www.apache.org/licenses/LICENSE-2.0
+*
+*Unless required by applicable law or agreed to in writing,
+*software distributed under the License is distributed on an
+*"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+*KIND, either express or implied.  See the License for the
+*specific language governing permissions and limitations
+*under the License.
+*/
+package org.wso2.am.integration.tests.header;
+
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.am.integration.test.utils.base.APIMIntegrationBaseTest;
+import org.wso2.am.integration.tests.header.util.SimpleSocketServer;
+import org.wso2.carbon.automation.engine.annotations.ExecutionEnvironment;
+import org.wso2.carbon.automation.engine.annotations.SetEnvironment;
+import org.wso2.carbon.automation.test.utils.http.client.HttpRequestUtil;
+import org.wso2.carbon.automation.test.utils.http.client.HttpResponse;
+
+import java.util.HashMap;
+
+public class APIMANAGER3614DuplicateTransferEncodingHeaderTestCase
+        extends APIMIntegrationBaseTest {
+
+    private SimpleSocketServer simpleSocketServer;
+
+    @BeforeClass(alwaysRun = true)
+    public void setEnvironment() throws Exception {
+        super.init();
+
+        apimTestCaseUtils.loadResource
+                ("/artifacts/AM/synapseconfigs/property/duplicate_transfer_encoding.xml");
```

Comment:
I'm getting a File Not Found exception due to this file. Seems you haven't added the file. Please update the pull request with it.